src/core/Sampler/Sampler.cpp

```diff
@@ -619,9 +669,10 @@ bool Sampler::renderNote( Note* pNote, unsigned nBufferSize, Song* pSong )
 			cost_L = cost_L * pSong->get_volume();	// song volume
 			cost_L = cost_L * 2; // max pan is 0.5
```

Comment:
[1] this factor over here? (and the same for the right channel)

Reply:
Good point! I didn't noticed it